content/en/vertical-scaling.md

```diff
@@ -2,7 +2,7 @@
 title: Vertical Scaling
 status: Completed
 category: Concept
-tags: ["infrastructure", "", ""]
+tags: ["infrastructure", "methodology", ""]
```

Comment:
Maybe not methodology?

Reply:
I think tags of `Vertical Scaling` and [horizontal-scaling.md](https://github.com/cncf/glossary/blob/main/content/en/horizontal-scaling) need to be similar to `Auto Scaling` because `Vertical Scaling` is sub-category of `Scaling`.

Currently, 

- suggested tags for `Auto Scaling` [property, infrastructure, architecture(?)]
- existing tag for [horizontal-scaling.md](https://github.com/cncf/glossary/blob/main/content/en/horizontal-scaling.md) [infrastructure]

---

Terms using `methodology` are as follows,

- agile-software-development.md
- blue-green-deployment.md
- canary-deployment.md
- chaos-engineering.md
- continuous-delivery.md
- continuous-deployment.md
- continuous-integration.md
- devops.md
- devsecops.md
- event-streaming.md
- security-chaos-engineering.md
- shift-left.md
- site-reliability-engineering.md
- version-control.md

Considering other terms using `methodology`, I think vertical-scaling doesn't fit.